docs/source/asr/configs.rst

```diff
@@ -752,7 +753,8 @@ Fine-tuning via a NeMo pretrained model name
         --config-name=<name of config without .yaml>) \
         model.train_ds.manifest_filepath="<path to manifest file>" \
         model.validation_ds.manifest_filepath="<path to manifest file>" \
-        trainer.gpus=-1 \
+        trainer.devices=-1 \
+        trainer.accelerator='gpu' \
```

Comment:
In the docs, there seems to be additional new line, upto you if you want to fix those (I would recommend to do it but there's a lot of files). If you can findreplace then try that c

Reply:
Probably we can look this next time, too many files already